plugins/inputs/dnac/dnac.go

```diff
@@ -0,0 +1,176 @@
+package dnac
```

Comment:
I recommend renaming this so it is easier identifiable seeing as this is the way Cisco seems to brand it looking it up online (e.g. https://www.cisco.com/site/us/en/products/networking/dna-center-platform/index.html). You will need to rename the package and folder names as well.

```suggestion
package cisco_dna_center
```